packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-settings/index.tsx

```diff
@@ -29,30 +29,30 @@ const extraSettingsStyles = css({
 });
 
 type PipelineSettingsProps = {
-  isSavePipelineEnabled?: boolean;
-  isCreatePipelineEnabled?: boolean;
+  isSavePipelineDisplayed?: boolean;
```

Comment:
Renamed because this was not really enabling them, but actually showing them on the screen or not and this was clashing with `isExportToLanguageEnabled`